.github/workflows/steward.yml

```diff
@@ -16,15 +16,15 @@ jobs:
         id: generate-token
         uses: tibdex/github-app-token@v1
         with:
-          app_id: 123456
+          app_id: ${{ secrets.STEWARD_APP_ID }}
           private_key: ${{ secrets.STEWARD_PRIVATE_KEY }}
 
       - name: Launch Scala Steward
         uses: scala-steward-org/scala-steward-action@v2
         with:
           github-token: ${{ steps.generate-token.outputs.token }}
           branches: series/0.23,main
-          author-email: ${{ secrets.STEWARD_APP_ID }}+typelevel-steward[bot]@users.noreply.github.com
+          author-email: 106827141+typelevel-steward[bot]@users.noreply.github.com
```

Comment:
That's annoying.  I had to do that in my Nix stuff, too.  And it opens dozens of PRs this way before we correct it.